tests/test_freqtradebot.py

```diff
@@ -238,143 +240,143 @@ def test_edge_overrides_stoploss(limit_buy_order, fee, caplog, mocker,
         assert trade.sell_reason == SellType.STOP_LOSS.value
 
 
-def test_total_open_trades_stakes(mocker, default_conf, ticker, fee) -> None:
+def test_total_open_trades_stakes(mocker, default_conf_usdt, ticker_usdt, fee) -> None:
     patch_RPCManager(mocker)
     patch_exchange(mocker)
-    default_conf['stake_amount'] = 0.00098751
-    default_conf['max_open_trades'] = 2
+    default_conf_usdt['stake_amount'] = 10.0
+    default_conf_usdt['max_open_trades'] = 2
     mocker.patch.multiple(
         'freqtrade.exchange.Exchange',
-        fetch_ticker=ticker,
+        fetch_ticker=ticker_usdt,
         get_fee=fee,
         _is_dry_limit_order_filled=MagicMock(return_value=False),
     )
-    freqtrade = FreqtradeBot(default_conf)
+    freqtrade = FreqtradeBot(default_conf_usdt)
     patch_get_signal(freqtrade)
     freqtrade.enter_positions()
     trade = Trade.query.first()
 
     assert trade is not None
-    assert trade.stake_amount == 0.00098751
+    assert trade.stake_amount == 10.0
     assert trade.is_open
     assert trade.open_date is not None
 
     freqtrade.enter_positions()
     trade = Trade.query.order_by(Trade.id.desc()).first()
 
     assert trade is not None
-    assert trade.stake_amount == 0.00098751
+    assert trade.stake_amount == 10.0
     assert trade.is_open
     assert trade.open_date is not None
 
-    assert Trade.total_open_trades_stakes() == 1.97502e-03
+    assert Trade.total_open_trades_stakes() == 20.0
 
 
-def test_create_trade(default_conf, ticker, limit_buy_order, fee, mocker) -> None:
+def test_create_trade(default_conf_usdt, ticker_usdt, limit_buy_order_usdt, fee, mocker) -> None:
     patch_RPCManager(mocker)
     patch_exchange(mocker)
     mocker.patch.multiple(
         'freqtrade.exchange.Exchange',
-        fetch_ticker=ticker,
+        fetch_ticker=ticker_usdt,
         get_fee=fee,
         _is_dry_limit_order_filled=MagicMock(return_value=False),
     )
 
     # Save state of current whitelist
-    whitelist = deepcopy(default_conf['exchange']['pair_whitelist'])
-    freqtrade = FreqtradeBot(default_conf)
+    whitelist = deepcopy(default_conf_usdt['exchange']['pair_whitelist'])
+    freqtrade = FreqtradeBot(default_conf_usdt)
     patch_get_signal(freqtrade)
-    freqtrade.create_trade('ETH/BTC')
+    freqtrade.create_trade('ETH/USDT')
 
     trade = Trade.query.first()
     assert trade is not None
-    assert trade.stake_amount == 0.001
+    assert trade.stake_amount == 10.0
     assert trade.is_open
     assert trade.open_date is not None
     assert trade.exchange == 'binance'
 
     # Simulate fulfilled LIMIT_BUY order for trade
-    trade.update(limit_buy_order)
+    trade.update(limit_buy_order_usdt)
 
-    assert trade.open_rate == 0.00001099
-    assert trade.amount == 90.99181073
+    assert trade.open_rate == 2.0
+    assert trade.amount == 30.0
```

Comment:
i have no idea whats happening here

Reply:
The problem here is the following:
the values passed in (stake_amount=10,  rate=2) would mean we'll be getting 5 ETH.
`limit_buy_order_usdt` however defines this order as `price=2.0, amount=30, cost=60` - assuming a stake_amount of 60$.

`trade.update()` will take the values from the order and update things.
real-world, this would happen for example on partially filled orders, or market orders where the price is unknown.
Now as the order does not align with the config (order defines a stake_amount of 60$, config of 10$) - we get strange values here.

Reply:
> The problem here is the following: the values passed in (stake_amount=10, rate=2) would mean we'll be getting 5 ETH. `limit_buy_order_usdt` however defines this order as `price=2.0, amount=30, cost=60` - assuming a stake_amount of 60$.
> 
> `trade.update()` will take the values from the order and update things. real-world, this would happen for example on partially filled orders, or market orders where the price is unknown. Now as the order does not align with the config (order defines a stake_amount of 60$, config of 10$) - we get strange values here.
> 
> I think an easy fix is to up stake_amount to 60 - using 10$ as stake can become a problem anyway, as realistically, you can't place orders for 10$ if you still wanna leave room for a stoploss below.

Sorry, I understood why it was a problem, but didn't understand why the test was passing

Reply:
The test is passing because we're checking the values that come out (30) - which does not correspond to the expeced result (due to the misalignment between order and stake-amount).

i've now updated stake-amount to 60$ - so it's aligned to the order, so this value makes sense again.